src/js/views/portals/PortalView.js

```diff
@@ -793,19 +805,22 @@ define([
         }
 
         if (!this.nodeView) {
-          //Update the location path with the new section name
-          this.updatePath(showSectionLabelInURL);
+          // Update the location path with the new section name
+          this.updatePath(
+            showSectionLabelInURL,
+            /* retainSearchQuery= */ !portalSectionView,
```

Comment:
It's not super obvious what use case !portalSectionView covers. Can we add a comment here?

Reply:
Done